.github/actions/verify-metrics-snapshot/action.yaml

```diff
@@ -55,16 +55,44 @@ runs:
         python3 -m pip install prometheus-client
         python3 ./scripts/e2e/compare_metrics.py --file1 ./.metrics/${{ inputs.snapshot }}.txt --file2 ./.metrics/baseline_${{ inputs.snapshot }}.txt --output ./.metrics/diff_${{ inputs.snapshot }}.txt
         if [ $? -eq 1 ]; then
-        echo "🛑 Differences found in metrics"
-        exit 1
+          echo "DIFF_FOUND=true" >> $GITHUB_OUTPUT
         fi
 
+    - name: Generate metrics summary
+      id: generate-summary
+      if: ${{ steps.generate-diff.outputs.DIFF_FOUND == 'true' }}
+      shell: bash
+      run: |
+        python3 ./scripts/e2e/metrics_summary.py \
+          --base ./.metrics/baseline_${{ inputs.snapshot }}.txt \
+          --pr ./.metrics/${{ inputs.snapshot }}.txt \
+          --output ./.metrics/summary_${{ inputs.snapshot }}.md
```

Comment:
please induce a temporary change in the metrics (e.g. change the name somewhere) so that we can see this workflow in action on this PR.